specification/src/main/asciidoc/platform/ServiceProviderInterface.adoc

```diff
@@ -1,64 +1,62 @@
 == Service Provider Interface
 
-The Java™ Platform, Enterprise Edition (Java
-EE) includes several technologies that are primarily intended to be used
-to extend the capabilities of the Java EE containers. In addition, some
-Java EE technologies include service provider interfaces along with
-their application programming interfaces. A Java EE profile may include
-some or all of these facilities, as described in
-link:#a3212[See Profiles]”.
+The Jakarta™ EE Platform includes several technologies that are primarily
+intended to be used to extend the capabilities of the Jakarta EE containers.
+In addition, some Jakarta EE technologies include service provider interfaces
+along with their application programming interfaces. A Jakarta EE profile may
+include some or all of these facilities, as described in link:#a3212[See Profiles]”.
 
-=== Java™ EE Connector Architecture
+=== Jakarta™ Connectors
 
 The Connector API defines how resource adapters
-are packaged and integrated with any Java EE product. Many types of
+are packaged and integrated with any Jakarta EE product. Many types of
 service providers can be provided using the Connector API and packaging,
-including JDBC drivers, JMS providers, and JAXR providers. All Java EE
-products must support the Connector APIs, as specified in the Connector
-specification.
+including JDBC drivers, Jakarta Messaging providers, and Jakarta XML Registries
+providers. All Jakarta EE products must support the Connector APIs, as specified
+in the Connector specification.
 
 The Connector specification is available at
 _http://jcp.org/en/jsr/detail?id=322_ .
 
-=== Java™ Authorization Service Provider Contract for Containers
+=== Jakarta™ Authorization
 
-The JACC specification defines the contract
-between a Java EE container and an authorization policy provider.
+The Jakarta Authorization specification defines the contract
+between a Jakarta EE container and an authorization policy provider.
 
-The JACC specification is available at
+The Jakarta Authorization specification is available at
```

Comment:
This link is actually for the old JACC specification...  Since we don't have the updated Jakarta Authorization Spec yet, we should probably revert this change to explain that the original JACC spec is available here...  Either that or just remove this reference in this portion of the doc.